scripts/build_schemas.sh

```diff
@@ -8,13 +8,18 @@ for region_code in GB-WLS GB-ENG GB-NIR; do
     FORMATTED_REGION_CODE=$(echo "${region_code}" | tr '[:upper:]' '[:lower:]' | tr - _)
     DESTINATION_FILE="data-source/jsonnet/census_individual_${FORMATTED_REGION_CODE}.json"
 
+    CENSUS_DATE="2019-10-13"
+    CENSUS_MONTH_YEAR_DATE="2019-10"
```

Comment:
```suggestion
    CENSUS_MONTH_YEAR_DATE=`echo $CENSUS_DATE | cut -f1,2 -d"-"`
```
To avoid duplication? Perhaps silly?

Reply:
Or are we better off having logic that covers date comparisons of this type (i.e. assume first of month when no day-of-month component of date)?

Reply:
I would say it better just duplicate one extra line instead of not nice looking sh commands.

Also regarding:

>   Or are we better off having logic that covers date comparisons of this type (i.e. assume first of month when no day-of-month component of date)?

I did speak to Maddaford about this and decided, for now, it was better/easier to just provide an extra month year date. But we should address it in future.